daemon/volumes_unix.go

```diff
@@ -129,9 +132,13 @@ func (daemon *Daemon) mountVolumes(container *container.Container) error {
 			return err
 		}
 
-		opts := "rbind,ro"
+		bindMode := "rbind"
```

Comment:
Would it be slightly clearer to use a variable for both "bind mode" and "write mode"? something like;

```suggestion
		bindMode := "rbind"
		if m.NonRecursive {
			bindMode = "bind"
		}

		writeMode := "ro"
		if m.Writable {
			writeMode = "rw"
		}

		opts := strings.Join([]string{bindMode, writeMode}, ",")
```